tambola/templates/public/manage.html

```diff
@@ -3,17 +3,29 @@
 {% extends "layout.html" %}
 {% block content %}
 <div class="row">
-    <div class="col-md-8">
+	<br>
+	<br>
+	<br>
+    <div class="col-md-8" align="center">
+		<div>
+	        <h1>Game {{game.id}}</h1>
 
-        <h1>Game {{game.id}}</h1>
-        <button type="button" class="btn btn-primary" onclick="nextNumber()">Generate</button>
-        <button type="button" class="btn btn-primary" onclick="newGame()">New Game</button>
-        {{game.numbers}}
+			<button type="button" class="btn btn-success btn-circle btn-xl" onclick="nextNumber()">Spin</button>
+		</div>
+		<br>
+		<div class="js-odoo"></div>
+		<div style="display: none" id="cur_num">{{game.numbers.split(",")[0]}}</div>
```

Comment:
does this cause issue when there arent any numbers?

Reply:
Nope, seems like it just ignores it.